package.json

```diff
@@ -24,7 +24,6 @@
   "peerDependencies": {
     "code-push": ">=1.8.0-beta",
     "cordova-plugin-file-transfer": ">=1.3.0",
-    "cordova-plugin-file": ">=3.0.0",
```

Comment:
I don't think they do, but it's worth double checking. I am fine with removing them completely - cordova should still be able to work without them. We might want to consider bumping the file plugin version in `config.xml` to take version of the plugin that contains the Android permissions fix.